src/components/publishing/sections/fullscreen_viewer/slide.tsx

```diff
@@ -0,0 +1,49 @@
+import { omit } from "lodash"
+import * as React from "react"
+import styled from "styled-components"
+import { pMedia } from "../../../helpers"
+import Fonts from "../../fonts"
+import Caption from "./caption"
+
+const Slide = props => {
+  const newProps = omit(props, ["section", "index", "total", "isCaptionOpen"])
+  const section = props.section
+  const src = section.url || section.image
+  return (
+    <div {...newProps}>
+      <SlideContainer>
+        <Title>{section.setTitle}</Title>
+        <Image src={src} />
```

Comment:
This should include the caption as alt text for accessibility

Reply:
👏 Best catch ever